plugins/analytics-module-segment/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@segment/backstage-plugin-analytics-module-segment",
+  "description": "Backstage analytics module plugin for Segment",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "private": false,
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "frontend-plugin-module",
+    "pluginId": "app",
+    "pluginPackage": "@backstage/plugin-app-backend"
+  },
```

Comment:
this config was stolen (with love) from the [backstage-community analytics plugins](https://github.com/backstage/community-plugins/tree/main/workspaces/analytics/plugins). This is what they set, so thats what I'll set.